### ISSUE TITLE
SAB and CAB

### ISSUE THREAD

**Issue by fceex49** [wurenkai/UltraLight-VM-UNet#18]
How important are SAB and CAB? I think in some other papers people just used simple bridge connection , which seems to work fine.

Not using SAB anf CAB would even further reduce the complexity and params/gflops?

**Comment by wurenkai** [comment]
SAB and CAB are mainly used to improve the fusion of multi-stage information in the skip-connection part, accelerating the model convergence and further improving the model performance. However, one value of 0.049M parametric quantities and 0.060GFLOPs cannot be achieved if they are solely relied upon. In addition, PVM Layer can be simply embedded into any model to replace the traditional Convolution, Transformers, Mamba and Vision Mamba, which can help your own model to significantly drop the parameters and GFLOPs. We have already tried more than 10 models, and in the future versions we will show more exciting experiments, and we are looking forward to your continuous attention.

**Comment by fceex49** [comment]
Do you have ablation results not using these two?

**Comment by wurenkai** [comment]
Yes, we tried it. There is a small drop in DSC in the absence of SAB and CAB (DSC of 0.9029). And as you say, this has already been proven to work in numerous previous experiments, mainly using with skip-connection paths, and is widely used. This is not our focus.
